templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1> Delete manufacturer</h1>
+  
+  <p>Are you sure want to delete the manufacturer: {{ manufacturer }}?</p>
+  <p><i>All cars with this manufacturer will be also deleted</i></p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger">
```

Comment:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on.